frontend/src/metabase/query_builder/components/filters/modals/BulkFilterList/utils.ts

```diff
@@ -57,3 +57,8 @@ const getSortValue = (dimensionOption: DimensionOption): number => {
 
 export const sortDimensions = (a: DimensionOption, b: DimensionOption) =>
   getSortValue(a) - getSortValue(b);
+
+export const isDimensionValid = (dimensionOption: DimensionOption) =>
+  dimensionOption.dimension.field().base_type === "type/Structured"
+    ? false
+    : true;
```

Comment:
What do you think about making this more concise?

```suggestion
  dimensionOption.dimension.field().base_type !== "type/Structured"
```